javascript-data-model/order-history.js

```diff
@@ -0,0 +1,93 @@
+var orderHistory = [
+  {
+    orderOneItems: [
+      {
+        placed: 'August 4, 2020',
+        total: 34.00,
+        shipsTo: 'JS Masher',
+        orderNum: 114 - 3941689 - 8772232,
+        dod: 'Aug 8, 2020',
+        title: 'JavaScript for impatient rogrammers',
+        author: 'Rauschmayer, Dr. Axel',
+        returnDue: 'Sep 7, 2020',
+        cost: 31.55,
+        reviewProd: true,
+        buyAgain: true,
+        viewItem: true,
+        prodSupport: false
```

Comment:
You have the data about the order itself mixed with data about a specific item, that should be separated. The object for order itself should have a property to store information about the items in the order, as well as other properties to store data about the order itself, such as the order number and who it was shipped to, etc.